src/main/java/duke/Task.java

```diff
@@ -0,0 +1,45 @@
+package duke;
+
+/**
+ * Represents a work that needs to be done.
+ * Contains a name of that work, represented by a String
+ * Contains the done status of the work. represented by a boolean.
+ */
+public class Task {
+    protected String taskName;
+    protected boolean isDone;
+
+    public Task(String taskName) {
+        this.taskName = taskName.trim();
+        this.isDone = false;
+    }
+
+    /**
+     * Visualises the done status of a task with an icon
+     *
+     * @return String version of check and cross, whichever needed
+     */
+    public String getStatusIcon() {
+        return isDone ?  "\u2713" : "\u2718";
+    }
+
+    public String getTaskName() {
+        return taskName;
+    }
+
+
```

Comment:
Very minor again, but maybe can delete the empty line.

Reply:
Got it, thanks!